### ISSUE TITLE
Bug: Streamwise messages not showing on zulip stat card to admin

### ISSUE THREAD

**Issue by codesankalp** [anitab-org/open-source-programs-web#72]
### Describe the bug
Streamwise messages are not showing on zulip stat to admin because the condition to check streamwise messages is wrong.
We need to delete this condition of checking all streamwise messages by checking the message in the newcomer stream,  as it is not required. To test the message

### To Reproduce
1. Go to http://localhost:8000/admin/osp/userinformation/
2. Edit one of the admin user-information and insert 345277 in zulip id.
3. Now go to this http://localhost:3000/ and log in using the credentials of the user which you have edited previously.
4. Reload the zulip stats and now you can see that streamwise messages do not show up.

### Expected behavior
Streamwise messages do not show up when the student has no message in the newcomer stream.

### Screenshots
![image](https://user-images.githubusercontent.com/56037184/107190877-649e0d00-6a11-11eb-9c68-92f6475b69e7.png)

### Desktop (please complete the following information):
 - OS: Ubuntu
 - Browser: Chrome
 - Version: _

### How to solve
To solve this issue remove the condition of checking the stream messages by checking message in newcomer stream.
https://github.com/anitab-org/open-source-programs-web/blob/ab18727af5a5e30829e4d07935a218c6d8a02762/src/components/Zulip.js#L39-L51

I have tested by removing it, it is working fine.

### Time Required to solve this issue
- 10 to 15 minutes.

**Comment by codesankalp** [comment]
@isabelcosta Can you please add labels to this issue?
As It is a beginner issue so a newcomer can solve this issue :relaxed:

**Comment by isabelcosta** [comment]
thank you @codesankalp for helping triage!

**Comment by prachinandi** [comment]
@isabelcosta  Ma'am if this issue is not assigned can I take it

**Comment by isabelcosta** [comment]
yes @prachi237 🤗 Will assign it to you!

**Comment by prachinandi** [comment]
Thanku ma'am😊

**Comment by codesankalp** [comment]
@prachi237 Any updates on this issue? Do you need any help or suggestion?

**Comment by prachinandi** [comment]
> @prachi237 Any updates on this issue? Do you need any help or suggestion?
😅sorry, for the delay I had exams, I started working on this issue again . Will send the PR asap

**Comment by codesankalp** [comment]
@prachi237 Is your exams over and can you make a PR on this?

**Comment by codesankalp** [comment]
Unassigning @prachi237 due to inactivity.

**Comment by sidharth3000** [comment]
if this is still available can i take this?

**Comment by Aaishpra** [comment]
Yess it is assigned you here, Do ask if you have any doubts

**Comment by sidharth3000** [comment]
project is not setting up, its giving me error.
<img width="960" alt="Screenshot 2021-05-20 231846" src="https://user-images.githubusercontent.com/55971223/119025602-ec58c780-b9c1-11eb-9612-598c85295eb3.png">

**Comment by codesankalp** [comment]
Have you setup the backend correctly and is it running?

**Comment by sidharth3000** [comment]
yes now its running, i had to bump up several dependencies. Now if i am not wrong i just have to remove this line. is it right?
![Uploading Screenshot 2021-05-21 172530.png…]()

**Comment by sidharth3000** [comment]
yes now its running, i had to bump up several dependencies. Now if i am not wrong i just have to remove this line. is it right?
<img width="851" alt="Screenshot 2021-05-21 172530" src="https://user-images.githubusercontent.com/55971223/119133635-c121b700-ba59-11eb-9433-48bf875aa911.png">

**Comment by vj-codes** [comment]
@sidharth3000 any updates?

**Comment by isabelcosta** [comment]
@sidharth3000 removed your assignment due to a lack of activity. Feel free to claim again, if you want to work on it

**Comment by msaini28r** [comment]
@isabelcosta can you assign this issue to me?

**Comment by isabelcosta** [comment]
@msaini0r , @codesankalp assigned this ticket to you :)

**Comment by msaini28r** [comment]
@isabelcosta please merge the pull request.

**Comment by ASP124** [comment]
Hello @codesankalp, I request you to grant me the access to work on this issue. 
Regards ASP

**Comment by AaryaDarisi** [comment]
Hey @codesankalp, @isabelcosta  i see that the issue is still open and left since a while and so i want to take this up if it is available but before picking it i wanted to ask if the steps mentioned in the HOW TO SOLVE section of the description is up to date coz i see that @msaini28r had followed it and ran into some issues. I wanted to confirm before starting to not to waste your time.
Thank you!

> @[msaini28r](https://github.com/msaini28r) msaini28r mentioned this issue [on Oct 15, 2021](https://github.com/anitab-org/anitab-forms-web/issues/72#ref-pullrequest-1027093873)
https://github.com/anitab-org/anitab-forms-web/pull/180
